x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/legend.tsx

```diff
@@ -38,17 +38,33 @@ export const JobMapLegend: FC<{ hasMissingJobNode: boolean; theme: EuiThemeType
   theme,
 }) => {
   const [showJobTypes, setShowJobTypes] = useState<boolean>(false);
+  const {
+    euiSizeM,
+    euiSizeS,
+    euiColorGhost,
+    euiColorWarning,
+    euiBorderThin,
+    euiBorderRadius,
+    euiBorderRadiusSmall,
+    euiBorderWidthThick,
+  } = theme;
 
   return (
-    <EuiFlexGroup
-      className="mlJobMapLegend__container"
-      alignItems="center"
-      data-test-subj="mlPageDataFrameAnalyticsMapLegend"
-    >
+    <EuiFlexGroup alignItems="center" data-test-subj="mlPageDataFrameAnalyticsMapLegend">
       <EuiFlexItem grow={false}>
         <EuiFlexGroup gutterSize="xs" alignItems="center">
           <EuiFlexItem grow={false}>
-            <span className="mlJobMapLegend__sourceNode" />
+            <span
+              data-test-subj="mlJobMapLegend__sourceNode"
+              css={{
+                height: `${euiSizeM}`,
```

Comment:
most of these overrides have mostly the same content. They could be pulled out into an object and reused.
e.g.something like
```
const cssOverrideBase = {
  height: `${euiSizeM}`,
  width: `${euiSizeM}`,
  backgroundColor: `${euiColorGhost}`,
  border: `${euiBorderWidthThick} solid ${theme.euiColorVis2}`,
  transform: 'rotate(45deg)',
  display: 'inline-block',
}
```
then used like this:
```
css={{
   ...cssOverrideBase
   transform: 'rotate(45deg)',
}}
```

also, it's not necessary to wrap single values in template literals

Reply:
Updated in 999a7c7c72865ff948b1f89091b944f539bed00b